Parser/pegen/pegen.c

```diff
@@ -2160,3 +2160,94 @@ _PyPegen_nonparen_genexp_in_call(Parser *p, expr_ty args)
         "Generator expression must be parenthesized"
     );
 }
+
+OperatorTermPair *
+_PyPegen_operator_term_pair(Parser *p, Token* oper, expr_ty factor)
+{
+    OperatorTermPair *a = PyArena_Malloc(p->arena, sizeof(OperatorTermPair));
+    if (!a) {
+        return NULL;
+    }
+    a->oper = oper;
+    a->factor = factor;
+    return a;
+}
+
+static operator_ty
+_PyPegen_get_binary_operator(const Token *t)
+{
+    switch (t->type) {
```

Comment:
Too bad this can't be a table.

Reply:
Interestingly, the compiler generates a table anyway :)

https://godbolt.org/z/zA9qhP